app/models/manageiq/providers/ibm_cloud/container_manager.rb

```diff
@@ -0,0 +1,201 @@
+class ManageIQ::Providers::IbmCloud::ContainerManager < ManageIQ::Providers::Kubernetes::ContainerManager
+  require_nested :Container
+  require_nested :ContainerGroup
+  require_nested :ContainerNode
+  require_nested :EventCatcher
+  require_nested :EventParser
+  require_nested :Refresher
+  require_nested :RefreshWorker
+
+  def self.ems_type
+    @ems_type ||= "iks".freeze
+  end
+
+  def self.description
+    @description ||= "IBM Cloud Kubernetes Service".freeze
+  end
+
+  def self.display_name(number = 1)
+    n_('Container Provider (IBM Cloud)', 'Container Providers (IBM Cloud)', number)
+  end
+
+  def self.default_port
+    443
+  end
+
+  def connect_options(options = {})
+    authentication = authentication_best_fit(options.fetch(:auth_type, "bearer"))
+    token = self.class.get_token(authentication.auth_key).id_token
+
+    super.merge(
+      :bearer => token,
+      :region => provider_region
```

Comment:
Do you actually need the region?  I don't see it passed to `self.class.get_token()`

Reply:
Yeah we don't need it, I missed that thanks